daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/crypto/Hash.scala

```diff
@@ -319,6 +321,29 @@ object Hash {
   ): Either[String, Hash] =
     handleError(assertHashContractKey(templateId, key))
 
+  private[this] def committedCidToBytes(coid: ContractId): Bytes =
+    coid match {
+      case coid: ContractId.V1 => coid.toBytes
```

Comment:
As I mentioned earlier, I'm in the process of figuring out what happens if we limit this to `coid.discriminator.bytes`. I want to put out a short doc about this by the end of the week. I don't mind merging this PR as-is though, as this hashing functionality won't be used elsewhere yet.

Reply:
I switched back the PR in draft mode while waiting for your conclusions